strictEqual.js

```diff
@@ -0,0 +1,11 @@
+const strictEqual = (a, b) => {
+
+  if (typeof(a) == typeof(b) && a == b) {
+    return true
+  }
+  else {
+    return false
+  }
```

Comment:
Here we're saying if the condition is true return true, if the condition is false return false. That means we can just return the condition since it will be the value we intend to return.

```JavaScript
const strictEqual = (a, b) => {
    return typeof a == typeof b && a == b
}
```

And since that would make it a one line return function we could use fat arrow's implicit return

```JavaScript
const strictEqual = (a, b) => typeof a == typeof b && a == b
```